views/play.erb

```diff
@@ -0,0 +1,9 @@
+
+<form method="get" action="/play">
+  <h1>Choose your first move:</h1>
+    <form method="get" action="/play">
+    <input type="submit" value="Rock">
+    <input type="submit" value="Paper">
+    <input type="submit" value="Scissors">
+</html>
```

Comment:
You have two <form> elements being opened but never being closed. Then a random </html> tag at the bottom of the page instead of a </form> tag. Maybe this page should be:

``` 
<h1>Choose your first move:</h1>
<form method="get" action="/play">
    <input type="submit" value="Rock">
    <input type="submit" value="Paper">
    <input type="submit" value="Scissors">
<form>
```